docs/development/core/server/kibana-plugin-server.savedobjectsdeleteoptions.md

```diff
@@ -0,0 +1,12 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [kibana-plugin-server](./kibana-plugin-server.md) &gt; [SavedObjectsDeleteOptions](./kibana-plugin-server.savedobjectsdeleteoptions.md)
+
+## SavedObjectsDeleteOptions interface
+
+
+<b>Signature:</b>
+
+```typescript
+export interface SavedObjectsDeleteOptions extends SavedObjectsBaseOptions, SavedObjectsMutatingOperationOptions 

```

Comment:
Our documentation tool isn't able to expand the properties of extended interfaces/classes https://github.com/microsoft/rushstack/issues/1075, so in the documentation `SavedObjectsDeleteOptions` ends up having no visible properties.

I don't expect you to fix up `SavedObjectsBaseOptions`, but if you could inline the refresh property from `SavedObjectsMutatingOperationOptions` then we're at least not moving further away from our docs reflecting the API.

Reply:
Ah, I didn't know about that limitation.
No problem, I was just trying to keep the types more concise.